src/plugins/PluginManager.ts

```diff
@@ -31,6 +32,18 @@ export class PluginManager {
         plugin.load(this.context);
     }
 
+    /**
+     * Update an event plugin
+     * @param config The config to update the plugin with.
+     */
+    public updatePlugin(pluginId: string, config: object) {
+        const plugin = this.plugins.get(pluginId);
+        switch (pluginId) {
```

Comment:
I left it as a `switch` statement in case we might want to update other plugins later on. If not needed or refactoring later is okay/trivial, then I can have the `PluginManager` call the `DomEventPlugin`'s `update` method directly.

Reply:
Is this in case `plugin.update` does not exist? In this case we can check if function exists before calling it. Something like what is done in the `record` function (except we probably don't want to throw an error).
```
if (plugin && plugin.record instanceof Function) {
    plugin.record(data);
}
```

Reply:
Refactored.